pkg/cluster/bootstrap.go

```diff
@@ -82,19 +85,138 @@ func (c *Cluster) shouldBootstrapLoad(ctx context.Context) (bool, error) {
+		c.config.Runtime.ControlRuntimeBootstrap.ETCDServerCAKey: "etcd/server-ca.key",
+		c.config.Runtime.ControlRuntimeBootstrap.ETCDPeerCA:      "etcd/peer-ca.crt",
+		c.config.Runtime.ControlRuntimeBootstrap.ETCDPeerCAKey:   "etcd/peer-ca.key",
+		c.config.Runtime.ClientETCDCert:                          "etcd/client.crt",
+		c.config.Runtime.ClientETCDKey:                           "etcd/client.key",
+		c.config.Runtime.PeerServerClientETCDCert:                "etcd/peer-server-client.crt",
+		c.config.Runtime.PeerServerClientETCDKey:                 "etcd/peer-server-client.key",
+		c.config.Runtime.ServerETCDCert:                          "etcd/server-client.crt",
+		c.config.Runtime.ServerETCDKey:                           "etcd/server-client.key",
+
+		c.config.Runtime.ClientAdminCert:                  "client-admin.crt",
+		c.config.Runtime.ClientAdminKey:                   "client-admin.key",
+		c.config.Runtime.ClientAuthProxyCert:              "client-auth-proxy.crt",
+		c.config.Runtime.ClientAuthProxyKey:               "client-auth-proxy.key",
+		c.config.Runtime.ClientCA:                         "client-ca.crt",
+		c.config.Runtime.ClientCAKey:                      "client-ca.key",
+		c.config.Runtime.ClientCloudControllerCert:        "client-cloud-controller.crt",
+		c.config.Runtime.ClientCloudControllerKey:         "client-cloud-controller.key",
+		c.config.Runtime.ClientControllerCert:             "client-controller.crt",
+		c.config.Runtime.ClientControllerKey:              "client-controller.key",
+		c.config.Runtime.ClientK3sControllerCert:          "client-k3s-controller.crt",
+		c.config.Runtime.ClientK3sControllerKey:           "client-k3s-controller.key",
+		c.config.Runtime.ClientKubeAPICert:                "client-kube-apiserver.crt",
+		c.config.Runtime.ClientKubeAPIKey:                 "client-kube-apiserver.key",
+		c.config.Runtime.ClientKubeletKey:                 "client-kubelet.key",
+		c.config.Runtime.ClientKubeProxyCert:              "client-kube-proxy.crt",
+		c.config.Runtime.ClientKubeProxyKey:               "client-kube-proxy.key",
+		c.config.Runtime.ClientSchedulerCert:              "client-scheduler.crt",
+		c.config.Runtime.ClientSchedulerKey:               "client-scheduler.key",
+		c.config.Runtime.ControlRuntimeBootstrap.ServerCA: "dynamic-cert.json", // ?
+		c.config.Runtime.RequestHeaderCA:                  "request-header-ca.crt",
+		c.config.Runtime.RequestHeaderCAKey:               "request-header-ca.key",
+		c.config.Runtime.ServerCA:                         "server-ca.crt",
+		c.config.Runtime.ServerCAKey:                      "server-ca.key",
+		c.config.Runtime.ServiceKey:                       "service.key",
+		c.config.Runtime.ServingKubeAPICert:               "serving-kube-apiserver.crt",
+		c.config.Runtime.ServingKubeAPIKey:                "serving-kube-apiserver.key",
+		c.config.Runtime.ServingKubeletKey:                "serving-kubelet.key",
+	}
```

Comment:
I think `c.config.Runtime.EncryptionConfig` also needs to be checked, if it's set in the bootstrap data.